src/Button.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Button = ({ onClick, className, children }) => (
+  <button
+    onClick={onClick}
+    className={className}
```

Comment:
This is the subversion little piece of `styled-components` that makes it hard to make a shared component like this!